backend/internal/storage/postgres/schema/review.go

```diff
@@ -0,0 +1,61 @@
+package schema
+
+import (
+	"context"
+	"fmt"
+	"platnm/internal/models"
+
+	"platnm/internal/errs"
+
+	"github.com/jackc/pgerrcode"
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/pgconn"
+	"github.com/jackc/pgx/v5/pgxpool"
+)
+
+type ReviewRepository struct {
+	*pgxpool.Pool
+}
+
+const (
+	UserFKeyConstraint = "review_user_id_fkey"
+)
+
+func (r *ReviewRepository) CreateReview(ctx context.Context, review *models.Review) (*models.Review, error) {
+	query := `
+	WITH media_exists AS (
+    SELECT CASE
+        WHEN $3 = 'track' AND EXISTS (SELECT 1 FROM track WHERE id = $2) THEN true
+        WHEN $3 = 'album' AND EXISTS (SELECT 1 FROM album WHERE id = $2) THEN true
+        ELSE false
+    END AS exists
+	)
+	INSERT INTO review (user_id, media_id, media_type, rating, comment)
+	SELECT $1, $2, $3::media_type, $4, $5
+	FROM media_exists
+	WHERE exists
+	RETURNING id;
+	`
+
+	if err := r.QueryRow(ctx, query, review.UserID, review.MediaID, review.MediaType, review.Rating, review.Comment).Scan(&review.ID); err != nil {
+		if err == pgx.ErrNoRows {
```

Comment:
```suggestion
		if errors.Is(err, pgx.ErrNoRows) {
```
prefer `if errors.Is(err, fs.ErrExist)` over `if err == fs.ErrExist`, per the [godocs](https://pkg.go.dev/errors)